src/SourceGenerators/Uno.UI.SourceGenerators/TSBindings/TSBindingsGenerator.cs

```diff
@@ -127,13 +140,15 @@ private static IEnumerable<INamedTypeSymbol> GetNamespaceTypes(IModuleSymbol mod
 			}
 		}
 
-		private int GetStructPack(INamedTypeSymbol parametersType)
+		private int GetStructPack(ISymbol parametersType)
 		{
 			// https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Symbols/TypeLayout.cs is not available.
 
-			if (parametersType.GetType().GetProperty("Layout", System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.NonPublic) is PropertyInfo info)
+			var actualSymbol = GetActualSymbol(parametersType);
+
+			if (actualSymbol.GetType().GetProperty("Layout", System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.NonPublic) is PropertyInfo info)
```

Comment:
Codacy found an issue: [Make sure that this accessibility bypass is safe here.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270956)